solar-land/scss/_variables.scss

```diff
@@ -0,0 +1,10 @@
+// _variables.scss
+
+// defining colors
+$primaryColor: #0dd47e; // Used for buttons, active states, and highlights.
```

Comment:
when you call it Primary Color, it means for the all part of the project it'll use, you don't need to comment it